### PR TITLE
Release 1.0 preparation

### PR DESCRIPTION
**Pull request by darosior**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,43 @@
...
+## 1.0
+
+This is the first non-beta release of Liana.
+
+Improvements were concentrated on the GUI. The UI was entirely overhauled.
+
+### Features
+
+- The `createspend` command now allows you to not provide any destination. In this case it will
+  create a self-to-send transaction containing a single change output.
```
Comment (jp1ac4) [inline CHANGELOG.md:12]: Should this be "send-to-self"?

Comment (darosior) [reply CHANGELOG.md:12]: Good spot, thanks!

**Comment by darosior** [comment]
Note to self: let's not forget to remove the `-dev` suffix after branch-off/

File: README.md [hunk 2]
```diff
@@ -14,22 +14,8 @@ Liana is a simple Bitcoin wallet that features a timelocked recovery path for al
...
-
-Liana is still under heavy development. Multisig support was implemented in the second release and
-the possibility for having multiple timelocked recovery paths was in the fourth release.  Regular
-wallet features (RBF, coin selection, ..) are planned. We also intend to switch to using Taproot as
-soon as possible, for enhanced privacy.
-
-**As such please consider Liana to be beta software.**
-
+Liana can be used for inheritance, decaying multisigs or safer backups. [Learn more about the
+usecases from our website](https://wizardsardine.com/liana).
```
Comment (darosior) [inline README.md:18]: The icon could link to the website too.

**Comment by darosior** [comment]
self-ACK fe906da6123d9741a6a1d3d8d94e6f632138e56f

We can continue improving the documentation in a follow-up.

**Pull request merged**